### DISCUSSION TITLE
Next button works in all cases except one

### DISCUSSION THREAD

**Discussion by raphroberts** [GetPublii/Publii#1627] [Help] [answered]
Hello, each of the pages of my site have a "next" button to go to the next page, but one page instead links to the home page:
https://content.une.edu.au/2023/oz_soils/soil-structure-2.html
I would expect the button to link to the "Hierarchy" page in this instance. Any help would be greatly appreciated. Thanks

**Comment by dziudek** [comment]
Hi @raphroberts - could you better describe your case? For me it looks correct - your homepage is https://content.une.edu.au/2023/oz_soils/ so I do not understand what's is wrong with link to https://content.une.edu.au/2023/oz_soils/oz-soils.html on https://content.une.edu.au/2023/oz_soils/soil-structure-2.html

Also which version of Publii are you using?

**Reply by raphroberts** [reply]
Hi @dziudek thank you for looking into this for me. I am using Publii version 0.45.2, build 16609.

Every page of this website has a small page-footer menu that links to the 'NEXT' and 'PREVIOUS' page.
I presumed (perhaps mistakenly?) that these links would be generated based upon the left-side menu. So this page for instance:

https://content.une.edu.au/2023/oz_soils/ants.html

lets you go forward to 'earthworms' or back to 'millipedes' which matches the tree of the left-side menu. The problem is that sometimes the link directs to the wrong place and I cannot determine why. I hope this makes sense :) Thanks again.

**Reply by raphroberts** [reply]
Hi again, I have noticed your pages have the same issue, e.g:

https://getpublii.com/dev/post-editor-api/

The "next" button at the bottom of this page links to:
https://getpublii.com/dev/text-input/

Where it should probably link to:
https://getpublii.com/dev/theme-settings-api/

It seems the links are based on page ID, so I guess the question is whether it is possible to control this order somehow, e.g by changing the ID of the pages somehow?

The alternative might be to try to write something that fetches the next page based on the left-side menu.

Hope this info helps. Thanks.

**Reply by dziudek** [reply, marked as answer]
@raphroberts - all is ok - the bottom navigation is totally unrelated to the menu on the left side and is just based on the posts ordering not the menu structure
